docs/topics/proc_configuring-your-application-to-use-spring-boot.adoc

```diff
@@ -47,6 +47,50 @@ Include the following properties in your `pom.xml` file to track the version of
 ----
 --
 
+. Specify the repositories containing the {SpringBoot} Maven plugin and the {SpringBoot} dependencies you are using in your application.
++
+[source,xml,subs="attributes+"]
```

Comment:
Suggestion: use `options="nowrap"` because we have some long lines here.

Reply:
Will add that, too. Thank you.